zerver/lib/bugdown/__init__.py

```diff
@@ -678,32 +678,43 @@ def handleMatch(self, match):
         img.set('alt', email)
         return img
 
-emoji_tree = os.path.join(settings.STATIC_ROOT, "generated", "emoji", "images", "emoji")
-path_to_emoji = os.path.join(emoji_tree, '*.png')
-path_to_unicode_emoji = os.path.join(emoji_tree, 'unicode', '*.png')
 path_to_name_to_codepoint = os.path.join(settings.STATIC_ROOT, "generated", "emoji", "name_to_codepoint.json")
 
-unicode_emoji_list = [os.path.splitext(os.path.basename(fn))[0] for fn in glob.glob(path_to_unicode_emoji)]
 name_to_codepoint = ujson.load(open(path_to_name_to_codepoint))
+unicode_emoji_list = [name_to_codepoint[name] for name in name_to_codepoint]
 
 
-def make_emoji(emoji_name, src, display_string):
-    # type: (Text, Text, Text) -> Element
+def make_emoji(codepoint, display_string):
+    # type: (Text, Text) -> Element
+    span = markdown.util.etree.Element('span')
+    span.set('class', 'emoji emoji-%s' % (codepoint,))
+    span.set('title', display_string)
+    return span
+
+def make_realm_emoji(src, display_string):
+    # type: (Text, Text) -> Element
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
     elt.set("title", display_string)
     return elt
 
+def to_codepoint(value):
```

Comment:
I would rename this to something like `unicode_emoji_to_codepoint`, and also use something more evocative than `value`. In the best case, someone reading just the `def` line and the mypy arguments can have a good guess at what the function does.